patches/pana.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/brick/__brick__/very_good_dart_package/.github/workflows/main.yaml b/brick/__brick__/very_good_dart_package/.github/workflows/main.yaml
```

Comment:
Thoughts on renaming to `pubspec_pana.patch` instead for consistency?

Reply:
This isn't a pubspec though, maybe `pana_workflow.patch`?

Reply:
SG, changed!